### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by StasOkhrym**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
-    # write your code here
-    pass
+    response = requests.get(URL, {"key": API_KEY, "q": FILTERING})
+    data = response.json()
+
+    if response.status_code == 200:
+        city = data["location"]["name"]
+        region = data["location"]["region"]
+        localtime = datetime.strptime(
+            data["location"]["localtime"], "%Y-%m-%d %H:%M"
```
Comment (rostyak) [inline app/main.py]: Do we need "%Y-%m-%d %H:%M"? API gives us formatted data

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+        )
+        temp_c = data["current"]["temp_c"]
+        last_updated = datetime.strptime(
+            data["current"]["last_updated"], "%Y-%m-%d %H:%M"
```
Comment (rostyak) [inline app/main.py]: Do we need "%Y-%m-%d %H:%M"? API gives us formatted data

**Review by rostyak — APPROVED** [review]

File: app/main.py (hunk 1 above)
Comment (Y-Havryliv) [reply app/main.py]: so why did you approve? @rostyak

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Y-Havryliv) [reply app/main.py]: rostyak is right, you can leave just `data["location"]["localtime"]`

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

**Review by Y-Havryliv — APPROVED** [review]

**Pull request closed without merging**